backend/src/app.js

```diff
@@ -2,12 +2,21 @@ const dotenv = require('dotenv');
 const express = require('express');
 const mongoose = require('mongoose');
 const bodyParser = require('body-parser');
+const https = require('https');
+const fs = require('fs');
+
+// Configure environment variable path
+require('dotenv').config({ path: `./.env.${process.env.NODE_ENV}` })
 
 const app = express();
-const port = 3000;
+const credentials = {
+	key: fs.readFileSync('key.pem'),
+	cert: fs.readFileSync('cert.pem'),
```

Comment:
Let's not hardcode these, can put the names as an env variable or in some utility JS file